docs/src/pages/examples/memory/memory-with-libsql.mdx

```diff
@@ -0,0 +1,105 @@
+# Memory with LibSQL
+
+This example demonstrates how to use Mastra's memory system with LibSQL as the storage backend.
+
+## Setup
+
+First, set up the memory system with LibSQL storage and vector capabilities:
+
+```typescript
+import { Memory } from '@mastra/memory';
+import { LibSQLStore } from '@mastra/store-libsql';
+import { LibSQLVector } from '@mastra/vector-libsql';
+import { Agent } from '@mastra/core/agent';
+
+// Initialize memory with LibSQL storage and vector search
+const memory = new Memory({
+  storage: new LibSQLStore({
+    url: process.env.DATABASE_URL || 'file:local.db',
+  }),
+  vector: new LibSQLVector({
+    url: process.env.DATABASE_URL || 'file:local.db',
+  }),
+  options: {
```

Comment:
`memoryOptions`?

Reply:
thread on this here https://github.com/mastra-ai/mastra/pull/1688/files#r1941531470